lua/starfall/libs_cl/render.lua

```diff
@@ -857,6 +857,17 @@ function render_library.setColor(clr)
 	surface.SetTextColor(clr)
 end
 
+--- Sets the draw color modulation. Note that, unlike the equivalent GLua function, this function takes numbers in a 0-255 range.
```

Comment:
Since this function is essentially scaling colors, it makes sense to take 0-1 instead of 0-255. Also would get rid of the optional Color table arg.

Reply:
Inputs to `render.setColorModulation` should definitely not be divided by `255`. You can use values higher than `1` to make colors on a texture bleed out or to make the hologram glow *(sort of)*.  

Red ranges from `1` to `1000` here *(without dividing by `255`)*:
![image](https://github.com/thegrb93/StarfallEx/assets/7283019/e24709d2-8dc9-4513-a000-f0ec8cb11da4)

Reply:
If you divide inputs of `render.setColorModulation` by `255`, then what about `render.setBlend`? It takes inputs in the `[0..1]` range right now. That's inconsistent.
 
I like the idea of having `render.setTint` that takes a color object and just calls `render.SetColorModulation` and `render.SetBlend` divided by `255` for you. Seems useful.